src/Admin/Settings/SettingsRegistry.php

```diff
@@ -678,37 +678,38 @@ public function show_forms() {
 	/**
 	 * Tabbable JavaScript codes & Initiate Color Picker
 	 *
-	 * This code uses localstorage for displaying active tabs
+	 * This code uses query arguments and localstorage for displaying active tabs
 	 *
 	 * @return void
 	 */
 	public function script() {
 		?>
 		<script>
 			jQuery(document).ready(function ($) {
-				//Initiate Color Picker
+				// Initiate Color Picker
 				$('.wp-color-picker-field').wpColorPicker();
 
 				// Switches option sections
 				$('.group').hide();
 				var activetab = '';
-				if (typeof (localStorage) != 'undefined') {
-					activetab = localStorage.getItem("activetab");
-				}
+				var urlParams = new URLSearchParams(window.location.search);
+				var queryTab = urlParams.get('tab');
 
-				//if url has section id as hash then set it as active or override the current local storage value
-				if (window.location.hash) {
```

Comment:
@josephfusco perhaps we should maintain this mechanism of using the hash? 

the deeplinking is technically already working. If I hover the tabs I can see the `#name-of-tab` in the url and if I copy the link and visit it, the link works 🤔 

I do think the improvments to store in localStorage and push to the url are good, but maybe instead of using `tab` query param, we can continue to use the hash in case anyone already is making use of linking with the hash

Reply:
Yes, great call out. I've updated this PR to use the existing hash functionality